### PR TITLE
feat: script for checking missing tokens

### PR DESCRIPTION
**Pull request by volfpe**
Added bin script to check the missing tokens in the supplied json file.

Example usage in package.json file that is using phoenix-components:
```
"scripts": {
  "build": "echo building && pc-check-tokens file.json"
}
```

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -45,6 +45,9 @@
 		"percy": "if [[ $(git rev-parse --symbolic-full-name --abbrev-ref HEAD) != \"master\" ]]; then percy storybook ./storybook-static; else echo \"ERROR: percy should not be run on master because these snapshots are auto-approved\"; fi;",
 		"test": "jest"
 	},
+	"bin": {
+		"pc-check-tokens": "./check-tokens/index.js"
```
Comment (samuelg0rd0n) [inline package.json:49]: Wouldn't it be better/cleaner to put this in a directory `bin`? There might be more scripts in the future so they can be grouped together in this folder. And also it's something that is widely understood (like `dist`, `src`, `build`) so the root folder has a clean structure. And the script itself can be named `check-tokens.js`. There's no need to name it `index` right? 😊

File: check-tokens/index.js [hunk 2]
```diff
@@ -0,0 +1,162 @@
+#!/usr/bin/env node
+
+const path = require('path')
+
+const keysToCheck = [
+	'borderWidth.focus',
```
Comment (samuelg0rd0n) [inline check-tokens/index.js:6]: And BTW, I need to check this list one more time. I think I have some items missing 😕 😃 But that's on me... I'll fix it in this PR.

File: package.json (hunk 1 above)
Comment (volfpe) [reply package.json:49]: good idea 👍

**Comment by volfpe** [comment]
@samuelg0rd0n zmeny za me ok

**Review by samuelg0rd0n — APPROVED** [review]

**Pull request merged**